pkg/inject/envoy.go

```diff
@@ -61,6 +69,22 @@ const envoyContainerTemplate = `
     {
       "name": "ENABLE_ENVOY_XRAY_TRACING",
       "value": "1"
+    }{{ end }}{{ if .EnableXrayTracing }},
+    {
+      "name": "XRAY_DAEMON_PORT",
+      "value": "{{ .XrayDaemonPort }}"
+    }{{ end }}{{ if .EnableDatadogTracing }},
+    {
+      "name": "ENABLE_ENVOY_DATADOG_TRACING",
+      "value": "1"
+    }{{ end }}{{ if .EnableDatadogTracing}},
+    {
+      "name": "DATADOG_TRACER_PORT",
+      "value": "{{ .DatadogTracerPort }}"
```

Comment:
We already configure the datadog address and port here [1], are these going to override the static resource config?


[1] https://github.com/aws/aws-app-mesh-controller-for-k8s/blob/master/pkg/inject/datadog.go#L29

Reply:
More like other way around but we should instead make the above config optional.

Reply:
After trying out both the modes, I believe we should stick to just passing the port and datadog tracer address. Default config that is being configured is same as what we send through the yaml. Also, this will give us the ability to configure the address and port instead of hardcoded values in the config file.